app/javascript/controllers/toggle_completed_controller.js

```diff
@@ -3,6 +3,7 @@ import { Controller } from "@hotwired/stimulus";
 export default class ToggleCompletedController extends Controller {
   connect() {
     this.completedTasks = document.querySelectorAll('.bg-success-subtle');
+    this.hideCompletedTasks();
```

Comment:
I prefer we hide completed pet tasks by default

Reply:
When I click to view the Tasks tab, the completed tasks are still displayed and the show completed toggle is off. I have to toggle on then back off for the completed tasks to be hidden.

Reply:
Can you double check your assets are loaded properly? Throw a console log in there and see it’s getting hit. I had to manually recompile to make it work (which is another issue to address but I thought would only be in docker)

Reply:
`rails assets:clobber` seemed to do the trick. Thanks